docs/usage.md

```diff
@@ -173,128 +261,215 @@ The pipeline also accepts a number of presets for common bisulfite library prepa
+`--num_mismatches` is `0.2` by default in Bismark, or `0.6` if `--relax_mismatches` is specified. `0.6` will allow a penalty of `bp * -0.6` - for 100bp reads, this is `-60`. Mismatches cost `-6`, gap opening `-5` and gap extension `-2`. So, `-60` would allow 10 mismatches or ~ 8 x 1-2bp indels.
 
 ### `--unmapped`
+
 Use the `--unmapped` flag to set the `--unmapped` flag with Bismark align and save the unmapped reads to FastQ files.
 
-### `--saveTrimmed`
+### `--save_trimmed`
+
 By default, trimmed FastQ files will not be saved to the results directory. Specify this flag (or set to true in your config file) to copy these files to the results directory when complete.
 
-### `--saveAlignedIntermediates`
-By default intermediate BAM files will not be saved. The final BAM files created after the deduplication step are always. Set to true to also copy out BAM files from the initial Bismark alignment step. If `--nodedup` or `--rrbs` is specified then BAMs from the initial alignment will always be saved.
+### `--save_align_intermeds`
+
+By default intermediate BAM files will not be saved. The final BAM files created after the deduplication step are always. Set to true to also copy out BAM files from the initial Bismark alignment step. If `--skip_deduplication` or `--rrbs` is specified then BAMs from the initial alignment will always be saved.
+
+### `--min_depth`
 
-### `--mindepth`
 Specify to specify a minimum read coverage for MethylDackel to report a methylation call.
 
-### `--ignoreFlags`
-Specify to run MethylDackel with the `--ignoreFlags` flag to ignore SAM flags.
+### `--meth_cutoff`
+
+Use this to specify a minimum read coverage to report a methylation call during Bismark's `bismark_methylation_extractor` step.
+
+### `--ignore_flags`
+
+Specify to run MethylDackel with the `--ignore_flags` flag to ignore SAM flags.
+
+### `--methyl_kit`
+
+Specify to run MethylDackel with the `--methyl_kit` flag to produce files suitable for use with the methylKit R package.
+
+### `--known_splices`
+
+Specify to run Bismark with the `--known-splicesite-infile` flag to run splice-aware alignment using HISAT2. A `.gtf` file has to be provided from which a list of known splicesites is created by the pipeline. (only works with `--aligner bismark_hisat`)
+
+### `--slamseq`
 
+Specify to run Bismark with the `--slam` flag to run bismark in [SLAM-seq mode](https://github.com/FelixKrueger/Bismark/blob/master/CHANGELOG.md#slam-seq-mode) (only works with `--aligner bismark_hisat`)
+
+### `--local_alignment`
+
+Specify to run Bismark with the `--local` flag to allow soft-clipping of reads. This should only be used with care in certain single-cell applications or PBAT libraries, which may produce chimeric read pairs. (See [Wu et al.](https://doi.org/10.1093/bioinformatics/btz125) (doesn't work with `--aligner bwameth`)
+
+## Job resources
 
-## Job Resources
 ### Automatic resubmission
```

Comment:
That would be good in the main nf-core docs

Reply:
It's in the template already. Feel free to make a PR to the website 😁